csrc/serde/fusion_cache.fbs

```diff
@@ -53,9 +56,12 @@ enum RecordType:int {
     CastVal,
     CatOp,
     ConstantBool,
-    ConstantInt,
-    ConstantDouble,
     ConstantComplexDouble,
+    ConstantComplexFloat,
+    ConstantDouble,
+    ConstantFloat,
```

Comment:
Any reason that we want to the constant variants, i.e. float/int/complexfloat?

Reply:
I didn't want to check the `dtype` field to determine if its subtype.

Reply:
Before:
```python
if (complex_double):
 if (complex_double.dtype() == ComplexFloat):
  ...
 else:
  ...
```

After:
```python
if (complex_double):
  ...
elif (complex_float):
  ...
```